src/api/api.py

```
@@ -0,0 +1,49 @@
+    questions: List[str]
+
+@app.post("/datasets/upload", response_model=DatasetResponse, tags=["Datasets"])
+async def upload_dataset(file: UploadFile):
+    # 1. Сохраняем файл
+    dataset_id = uuid4()
+    # 2. Пишем в БД Postgres (Dataset status: anonymizing)
+    # 3. Отправляем в RabbitMQ (celery_app.send_task("anonymize", ...))
+    return {"dataset_id": dataset_id, "status": "anonymizing"}
+
+@app.get("/datasets/{id}", tags=["Datasets"])
+async def get_dataset_status(id: UUID):
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)